.eslintrc.cjs

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright 2020 - 2024 Pionix GmbH and Contributors to EVerest
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
```

Comment:
alternatively we could also just drop the copyright year (range) (I'm not a lawyer, but there's precedent in multiple big open source projects for that)

Reply:
Yeah, but isn't this a general EVerest decision we should discuss for all repos? We use the spdx identifier everywhere. This is just the only project which uses a license scanner end therefore enforces that it is correct across all files :D